Assets/Drag.cs

```diff
@@ -10,23 +10,52 @@ public class Drag : MonoBehaviour
     Vector3 startingPosition;
     GameObject clone;
     Boolean triggerFlag;
+    Boolean enableFlag;
+
+    public Console console;
+
+    private MeshRenderer renderer;
+    private MeshRenderer clone_renderer;
```

Comment:
Change `clone_preview` to camel case instead of snake, should just be a rename function in VS or crtl + R